solar_system_wave3.rb

```diff
@@ -0,0 +1,160 @@
+# Maria I Velez - Octos - C9
+# 12 Feb 2018
+# Solar System program
+# https://github.com/Ada-C9/Solar-System
+
+### WAVE 3
+
+# Create a SolarSystem class with an @planets instance variable.
+class SolarSystem
+  # Create an initialize method which should take a collection of planet names and store them in an @planets instance variable.
+  def initialize(all_planets)
+    @planets = all_planets
+  end
+
+  attr_accessor :planets
+
+  # Create a method that adds a planet to the list (not using user input)
+  def planets
+    return @planets
+  end
+
+  # Create a method which will return, not print, a list of the planets as a String in this style
+  def summary
+    counter = 0
+    temp_list = []
+    @planets.each do | planet |
+      counter += 1
+      temp_list.push("#{counter}. #{planet.name}")
+    #   puts "#{planet[:name]}"  <-- this is what I used for Wave 1
+    puts "#{counter}. #{planet.name}"
+    end
+    return temp_list
```

Comment:
You both `puts` and `return` the planet names here - you should probably choose one or the other.